src/main/resources/infernal-expansion.mixins.json

```diff
@@ -21,7 +21,8 @@
     "common.MixinShearsItem",
     "common.MixinShovelItem",
     "common.MixinStriderEntity",
-    "common.WorldCarverAccessor"
+    "common.WorldCarverAccessor",
+    "common.MixinIngredientAccessor"
```

Comment:
Also, keep naming convention consistent. It looks there like the Accessors don’t have Mixin on the class name, like WorldCarverAccessor.

Reply:
I saw that but missed the fact it didnt include mixin, ill change that